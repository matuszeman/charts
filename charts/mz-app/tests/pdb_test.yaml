suite: pdb
values:
  - ../values-ci.yaml
templates:
  - pdb.yaml
tests:
  - it: should match snapshot with values-ci.yaml
    asserts:
      - matchSnapshot: {}
