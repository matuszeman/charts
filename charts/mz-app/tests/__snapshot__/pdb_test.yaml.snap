should match snapshot with values-ci.yaml:
  1: |
    apiVersion: policy/v1
    kind: PodDisruptionBudget
    metadata:
      name: RELEASE-NAME-resource-suffix
    spec:
      maxUnavailable: 0
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: app-name
